react-native/react/menubar/index.render.desktop.js

```diff
@@ -148,38 +154,60 @@ class CollapsableFolderList extends Component {
     folders: Array<FolderInfo>,
     folderDisplayLimit: number,
     collapsed: boolean,
-    onExpand: () => void
+    onExpand: Function,
+    isPublic: boolean
   };
 
   render () {
-    const {collapsed, onExpand, username, folderDisplayLimit} = this.props
-    const folderToElement = f => <FolderRow key={f.folderName} username={username} folder={f}/>
+    const {collapsed, username, folderDisplayLimit, onExpand} = this.props
 
     let {folders} = this.props
-    let truncatedCount = 0
     // Check if it's bigger by one because it's pointless to have a button
     // that says show all for just one more thing
+    let truncated = false
     if (collapsed && folders.length > folderDisplayLimit + 1) {
       folders = folders.slice(0, folderDisplayLimit)
-      truncatedCount = this.props.folders.length - folderDisplayLimit
+      truncated = true
     }
 
     return (
-      <div style={{...globalStyles.flexBoxColumn}}>
-        {intersperseFn(i => <Divider key={i} />, folders.map(folderToElement))}
-        {truncatedCount > 0 && <Button primary onClick={onExpand} label={`Show all (+${truncatedCount})`} style={{alignSelf: 'center'}}/>}
+      <div style={{...globalStyles.flexBoxColumn, marginLeft: 10, marginBottom: 4, justifyContent: 'flex-start'}}>
+        {folders.map(f => {
+          const key = f.type === 'entry' ? `entry${f.prefix}` : `${f.isPublic}:${f.folderName}`
+
+          if (f.type === 'entry') {
+            return <FolderEntry key={key} entry={f} />
+          } else {
+            return <FolderRow key={key} username={username} folder={f}/>
+          }
+        })}
+        {truncated && <ShowAll onClick={onExpand} isPublic={this.props.isPublic} />}
       </div>
     )
   }
 }
 
+const Version = props => {
+  // $FlowIssue ignore
+  const version = __VERSION__ // eslint-disable-line no-undef
+  return (
+    <Text
+      type='Body' small link style={{alignSelf: 'flex-end', fontSize: 10}}
+      onClick={() => {
+        clipboard.writeText(`Keybase GUI Version: ${version}`)
+        shell.openExternal('https://github.com/keybase/client/issues')
+        new Notification('Version copied to clipboard') //eslint-disable-line
```

Comment:
You should use: https://github.com/keybase/client/blob/nojima/DESKTOP-269-new-menu/react-native/react/native/notifications.desktop.js#L4